monai/networks/layers/factories.py

```diff
@@ -263,8 +265,21 @@ def instance_nvfuser_factory(dim):
     if dim != 3:
         warnings.warn(f"`InstanceNorm3dNVFuser` only supports 3d cases, use {types[dim - 1]} instead.")
         return types[dim - 1]
-    if not has_nvfuser:
-        warnings.warn("`apex.normalization.InstanceNorm3dNVFuser` is not found, use nn.InstanceNorm3d instead.")
+    # test InstanceNorm3dNVFuser installation with a basic example
+    has_nvfuser_flag = has_nvfuser
+    if not torch.cuda.is_available():
+        has_nvfuser_flag = False
```

Comment:
if no cuda could we return the regular class after line 271?

Reply:
Thanks, I resolved it.